api/buildings/management/commands/buildings.py

```diff
@@ -0,0 +1,106 @@
+        if options["delete"]:
+            buildings_count = Building.objects.count()
+            Building.objects.filter(
+                general_id__in=list(
+                    Building.objects.values_list("pk", flat=True)[
+                        : int(options["total_buildings"])
+                    ]
+                )
+            ).delete()
+            self.stdout.write(
+                self.style.SUCCESS(
+                    "Successfully deleted %s building(s)."
+                    % (int(buildings_count))
+                )
+            )
+        elif options["create"]:
+            for building_id in trange(int(options["total_buildings"])):
+                address = fake.address()
+                risk_category = random.choice(
+                    ["U1", "U2", "U3", "U4", "RS1", "RS2", "RS3", "RS4"]
+                )
+                county = random.choice(
+                    [
+                        "Bucharest",
+                        "Iasi",
+                        "Constanta",
+                        "Cluj",
+                        "Timisoara",
+                        "Brasov",
+                        "Sibiu",
+                    ]
+                )
+                post_code = random.choice(["020244", "905600"])
+                locality = random.choice(["1", "2", "3", "4", "5", "6"])
+                status = random.choice(["-1", "0", "1"])
+                certified_expert = fake.first_name() + " " + fake.last_name()
+                examination_year = fake.date("%Y")
+                lat = fake.latitude()
+                lng = fake.longitude()
+                year_built = fake.date("%Y")
+                apartment_count = fake.random_int(min=0, max=199)
+
+                new_building = Building.objects.create(
+                    address=address,
+                    risk_category=risk_category,
+                    county=county,
+                    post_code=post_code,
+                    locality=locality,
+                    status=status,
+                    certified_expert=certified_expert,
+                    examination_year=examination_year,
+                    lat=lat,
+                    lng=lng,
+                    year_built=year_built,
+                    apartment_count=apartment_count,
+                )
+
+                new_building.save()
+                sleep(0.01)
+            self.stdout.write(
+                self.style.SUCCESS(
+                    "Successfully created %s building(s)." % (building_id + 1)
+                )
+            )
+        else:
+            self.stdout.write(
+                self.style.ERROR(
+                    "Missing required argument: --create or --delete"
+                )
+            )
+            self.stdout.write(
+                self.style.SUCCESS(
+                    "Try: python manage.py buildings %s --create"
+                    % (int(options["total_buildings"]))
+                )
+            )
```

Comment:
Not sure if this is supposed to be here

Reply:
It's just a suggestion with the correct command.